PluralKit.Core/Migrations/6.sql

```diff
@@ -0,0 +1,23 @@
+-- Create new columns --
```

Comment:
Would like date at the migration header (although I can do this once merged)

Reply:
This and the changelog for the api docs would need to be updated, i'll stick in a temporary date though